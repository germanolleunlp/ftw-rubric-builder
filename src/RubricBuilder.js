import React, { PureComponent } from 'react';

import styles from './RubricBuilder.scss';

export class RubricBuilder extends PureComponent {
  render() {
    return (
      <div className={styles['rubric-builder-container']}>
        <p>Comming soon...</p>
      </div>
    );
  }
}
