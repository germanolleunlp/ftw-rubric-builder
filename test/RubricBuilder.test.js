import React from 'react';
import { shallow } from 'enzyme';

import { RubricBuilder } from '../src/RubricBuilder';

describe('RubricBuilder', () => {
  it('should render correctly', () => {
    const component = shallow(<RubricBuilder />);
    expect(component).toMatchSnapshot();
  });
});
