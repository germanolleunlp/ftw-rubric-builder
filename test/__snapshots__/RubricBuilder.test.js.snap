// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RubricBuilder should render correctly 1`] = `
<div>
  <p>
    Comming soon...
  </p>
</div>
`;
